examples/messaging/Cargo.toml

```diff
@@ -14,3 +14,7 @@ serde = { version = "1.0", features = ["derive"] }
 
 [dev-dependencies]
 mantle-test = { git = "https://github.com/oasislabs/mantle" }
+
+[patch."https://github.com/oasislabs/mantle"]
```

Comment:
was this for local testing?

Reply:
yes it's to get CI to pass so that I can do a switcharoo once you see the green checkmark